src/modules/class-wpml-elementor-multiple_gallery.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * WPML_Elementor_Multiple_Gallery
+ */
+namespace WPML\PB\Elementor\Modules;
+
+class WPML_Elementor_Multiple_Gallery extends \WPML_Elementor_Module_With_Items {
+
+    protected function get_title($field)
+    {
+        switch ( $field ) {
+            case 'gallery_title':
+                return esc_html__( 'Gallery Title:', 'sitepress' );
+            default:
+                return '';
+        }
+    }
+
+    protected function get_fields()
```

Comment:
@GeorgeBotsev, I think there's something wrong with this method because it's declared as an abstract "protected" method in the parent class and I see in other child classes we use it as a "public" method (and thus we have a test on it).

I don't want to make changes on the parent class right now because we have plans to extract all this config in a near future. But let's change here to "public" as we do in other similare child classes and your tests should pass then.

Reply:
I think I just changed this prior toasking for a review.